src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const sum = (a) => (b) => {
```

Comment:
Let's use more readable syntax. And yes, Ulana is correct, no need to create a variable, if you return it right away.
```suggestion
  return (a) => {
    return (b) => {
```